EuclideanGeometry/Foundation/Axiom/Basic/Angle/FromMathlib.lean

```diff
@@ -734,3 +734,358 @@ theorem AngValue.arg_expMapCircle (θ : AngValue) :
 end EuclidGeom
 
 end Mathlib.Analysis.SpecialFunctions.Complex.Circle
+
+section Mathlib.Data.Complex.Exponential
+/-
+Translate useful theorems in `Mathlib.Data.Complex.Exponential` to `AngValue.sin` or `AngValue.cos`, such as the sum and difference formulas, product-to-sum formulas, and sum-to-product formulas and so on.
+For example, the following theorems should be moved here.
```

Comment:
This file only contains lemmas that are directly translated from lemmas about `Real.Angle` in mathlib.

The lemmas in this PR are about `Real.sin` and `Real.cos`. Their `Real.Angle.sin` and `Real.Angle.cos` versions do not exist in mathlib. It feels like these lemmas should be put in another (new?) file. Maybe we can create some PRs to mathlib in the future.